api/public/index.php

```diff
@@ -25,7 +25,7 @@
 }
 
 if ($trustedProxies = $_SERVER['TRUSTED_PROXIES'] ?? false) {
-    Request::setTrustedProxies(explode(',', $trustedProxies), Request::HEADER_X_FORWARDED_ALL);
```

Comment:
Looks like an issue to me. You also need to keep the other `X_FORWARDED` headers.

Reply:
In fact it looks like `HEADER_X_FORWARDED_PROTO`  was ever set with `HEADER_X_FORWARDED_ALL` as the comment say.
```HEADER_X_FORWARDED_ALL = 0b11110; // All "X-Forwarded-*" headers```

Reply:
So i think the `mixed-content` is du to other stuff